### DISCUSSION TITLE
offline deploy open-webui

### DISCUSSION THREAD

**Discussion by LiuYuPeng1101** [open-webui/open-webui#1681] [General]
# Bug Report

## Description
I deployed open-webui offline, and the following situation occurred? Please tell me what I should do? Note, I am in an offline environment
![cbb7e57b4034bad74920b74fc6db753](https://github.com/open-webui/open-webui/assets/42868868/25ecfa62-bdf8-4b7c-8d52-440364b54aa4)

**Bug Summary:**
[Provide a brief but clear summary of the bug]

**Steps to Reproduce:**
[Outline the steps to reproduce the bug. Be as detailed as possible.]

**Expected Behavior:**
[Describe what you expected to happen.]

**Actual Behavior:**
[Describe what actually happened.]

## Environment

- **Open WebUI Version:** [e.g., 0.1.120]
- **Ollama (if applicable):** [e.g., 0.1.30, 0.1.32-rc1]

- **Operating System:** [e.g., Windows 10, macOS Big Sur, Ubuntu 20.04]
- **Browser (if applicable):** [e.g., Chrome 100.0, Firefox 98.0]

## Reproduction Details

**Confirmation:**

- [ ] I have read and followed all the instructions provided in the README.md.
- [ ] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.

## Logs and Screenshots

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots (if applicable):**
[Attach any relevant screenshots to help illustrate the issue]

## Installation Method

[Describe the method you used to install the project, e.g., manual installation, Docker, package manager, etc.]

## Additional Information

[Include any additional details that may help in understanding and reproducing the issue. This could include specific configurations, error messages, or anything else relevant to the bug.]

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by justinh-rahb** [comment]
Please follow the issue report template and provide some useful logs and information about your setup and environment so that someone may be able to assist.
